examples/demo-rollup/tests/test_helpers.rs

```diff
@@ -31,6 +39,49 @@ fn make_contract_from_abi(path: PathBuf) -> BaseContract {
     BaseContract::from(abi)
 }
 
+fn create_mock_da_rollup(rollup_config: RollupConfig<()>) -> Rollup<Risc0Verifier, MockDaService> {
+    let _ = remove_dir_all(&rollup_config.storage.path);
+    let ledger_db = initialize_ledger(rollup_config.storage.path.clone());
+    let sequencer_da_address = MockAddress { addr: [99; 32] };
+    let da_service = MockDaService::new(sequencer_da_address);
+
+    let app = App::new(rollup_config.storage);
+
+    let genesis_config = get_genesis_config(sequencer_da_address);
+
+    Rollup {
+        app,
+        da_service,
+        ledger_db,
+        runner_config: rollup_config.runner,
+        genesis_config,
+    }
+}
+
+pub async fn start_rollup(rpc_reporting_channel: oneshot::Sender<SocketAddr>) {
+    let mut mock_path = PathBuf::from("tests");
+    mock_path.push("test_data");
+    mock_path.push("tmp");
+    mock_path.push("mocks");
```

Comment:
I think the tests fail here https://github.com/Sovereign-Labs/sovereign-sdk/actions/runs/5944938901/job/16123134157?pr=716#step:7:132 because we share the same db path. Let's add another string argument, `test_name`, and push it to the `mock_path` this way tests will have a unique db.

Reply:
I switched to using `tempfile` instead, both to avoid needing to pick an argument value and for consistency with other tests.